web/client/plugins/tutorial/selectors.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2017, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const selectors = {
```

Comment:
Could it be like this? Or we always have to declare the preset

preset home
```
{
 "name": "Tutorial",
 "cfg": {
  "preset": "home"
 }
}
```

preset map 

```
{
 "name": "Tutorial"
}
```